src/v2/guide/migration.md

```diff
@@ -684,45 +684,45 @@ You can use:
 </div>
 {% endraw %}
 
-## Custom Directives <sup>simplified</sup>
+## Directivas personalizadas <sup>simplificado</sup>
 
-Directives have a greatly reduced scope of responsibility: they are now only used for applying low-level direct DOM manipulations. In most cases, you should prefer using components as the main code-reuse abstraction.
+Las directivas tienen un alcance de responsabilidad muy reducido: ahora solo se usan para aplicar manipulaciones del DOM de bajo nivel. En la mayoría de los casos, debería preferir usar componentes como la abstracción principal de reutilización de código.
 
-Some of the most notable differences include:
+Algunas de las diferencias más notables incluyen:
 
-- Directives no longer have instances. This means there's no more `this` inside directive hooks. Instead, they receive everything they might need as arguments. If you really must persist state across hooks, you can do so on `el`.
-- Options such as `acceptStatement`, `deep`, `priority`, etc have all been removed. To replace `twoWay` directives, see [this example](#Two-Way-Filters-replaced).
-- Some of the current hooks have different behavior and there are also a couple new hooks.
+- Las directivas ya no tienen instancias. Esto significa que no hay más `this` dentro de los ganchos de la directiva. En cambio, reciben todo lo que puedan necesitar como argumentos. Si realmente debe persistir el estado a través de los ganchos, puede hacerlo en `el`.
+- Opciones como `acceptStatement`, `deep`, `priority`, etc han sido removidas. Para reemplazar directivas `twoWay`, vea [este ejemple](#Two-Way-Filters-replaced).
+- Algunos de los ganchos actuales tienen un comportamiento diferente y también hay un par de ganchos nuevos.
```

Comment:
What do you think about:

```suggestion
- Algunos de los ganchos actuales tienen un comportamiento diferente y también hay un algunos ganchos nuevos.
```

Reply:
@FLarra @Tintef  In other places we are using `hooks` instead of `ganchos`. What do you think about use `hooks`?